src/services/appCRUD/appConfService.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Cognite AS
+import { MESSAGES } from 'constants/messages';
+import {
+  getMachineConfig,
+  saveMachineConfig,
+} from './appConfRepo/appConfFSRepo';
```

Comment:
Isn't there a better way to implement polymorphism in JS? Currently, service class is coupled with actual implementation.   Ideally it has to be decoupled and storage API must be injected to this some how.

Reply:
I followed the same way we save the widget configurations and there also this issue has been raised. Asanka has raised a task (#96 ) for that and we'll cover both scenarios under that task